.Rprofile

```diff
@@ -83,50 +76,23 @@ if (!dir.exists(".venv/")
 # REMIND_repos_dirs="C:\my\first\path;D:\my\second\path"
 remindReposDirs <- Sys.getenv("REMIND_repos_dirs")
 
-# for scp targets, you need to set three environment variables
-# on linux e.g. (separate multiple paths by semicolons)
-# REMIND_repos_scp="scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output;scp://cluster.pik-potsdam.de/p/projects/remind/inputdata/CESparametersAndGDX"
-# REMIND_repos_scp_user="myusername"  # use your user name on the scp target, e.g. the cluster
-# REMIND_repos_scp_key="/home/myusername/.ssh/id_ed25519"  # path to your your ssh private key on your laptop
-# on windows e.g.
-# REMIND_repos_scp="scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output;scp://cluster.pik-potsdam.de/p/projects/remind/inputdata/CESparametersAndGDX"
-# REMIND_repos_scp_user="myusername"  # use your user name on the scp target, e.g. the cluster
-# REMIND_repos_scp_key="C:\Users\myusername\.ssh\id_ed25519"  # path to your your ssh private key on your laptop
-remindReposSCP <- Sys.getenv("REMIND_repos_scp") # scp URL
-remindReposSCPUser <- Sys.getenv("REMIND_repos_scp_user")  # ssh user name
-remindReposSCPKey <- Sys.getenv("REMIND_repos_scp_key")  # ssh key path
-
 # unless specified otherwise, use cluster defaults
 use_cluster_defaults <- TRUE
 
 # add local directories, if any
 if ("" != remindReposDirs) {
-  directories <- unlist(strsplit(remindReposDirs, .Platform$path.sep,
-                                 fixed = TRUE))
+  directories <- unlist(strsplit(remindReposDirs, .Platform$path.sep, fixed = TRUE))
   directoriesList <- rep(list(NULL), length(directories))
   names(directoriesList) <- directories
-
   options(remind_repos = c(options("remind_repos")[[1]], directoriesList))
   use_cluster_defaults <- FALSE
 }
 
-# add remote directories, if any remote directory and username and SSH key are set
-if ("" != remindReposSCP && "" != remindReposSCPUser && "" != remindReposSCPKey) {
-  SCPUrls <- unlist(strsplit(remindReposSCP, ";", fixed = TRUE))
-  config <- list(list(username = remindReposSCPUser, ssh_private_keyfile = remindReposSCPKey))
-  for (SCPUrl in SCPUrls) {
-    names(config) <- SCPUrl
-    options(remind_repos = c(options("remind_repos")[[1]], config))
-  }
-  use_cluster_defaults <- FALSE
-}
-
 # default to cluster directories
-if (use_cluster_defaults &&
-    all(file.exists(c("/p/projects/rd3mod/inputdata/output",
-                      "/p/projects/remind/inputdata/CESparametersAndGDX")))) {
+if (use_cluster_defaults && all(file.exists(c("/p/projects/rd3mod/inputdata/output_1.27",
+                                              "/p/projects/remind/inputdata/CESparametersAndGDX")))) {
   options(remind_repos = list(
-    "/p/projects/rd3mod/inputdata/output" = NULL,
+    "/p/projects/rd3mod/inputdata/output_1.27" = NULL,
```

Comment:
I just discovered, that also for REMIND the correct path to `output_1.27` is already set when logging in to the new cluster. At least if you have set up your `.profile` correctly as recommended [here in the tutorial](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Cluster-Access#modules), because then this `/p/projects/rd3mod/R/.Rprofile_1.27` is loaded, as it is linked in line 27 here `/p/system/modulefiles/defaults/piam/1.27`. The latter file is probably loaded by line 9 
```
VERSION=${1:-"1.27"}
module load base-env "piam/$VERSION"
```
in this file `/p/system/modulefiles/defaults/piam/module_load_piam`, which is sourced from your `.profile`.

Do you agree?

Reply:
I always seem to forget about `.Rprofile_1.27`. Then we can remove the [whole block](https://github.com/remindmodel/remind/blob/f048a79d6f6fadc9bb87cef0016f41350f3552f0/.Rprofile#L91-L98), right?

After `source /p/system/modulefiles/defaults/piam/module_load_piam` and starting a R session I get 

```
> getOption("remind_repos")
$`/p/projects/rd3mod/inputdata/output`
NULL

$`/p/projects/rd3mod/inputdata/output_1.27`
NULL

$`/p/projects/remind/inputdata/CESparametersAndGDX`
NULL
```

Should `/p/projects/rd3mod/inputdata/output` even be in there?

Reply:
I would say yes. [This function loops](https://github.com/pik-piam/gms/blob/43226fbee337d25885649a24c5128646002a4552/R/download_unpack.R#L61) over the repos in the given order. If it does not find the input date in `/output` it will continue with `/output_1.27`.